apps/web/components/PageWrapper.tsx

```diff
@@ -27,6 +30,23 @@ const calFont = localFont({
   weight: "600",
 });
 
+const Redirects = () => {
+  const session = useSession();
+  const router = useRouter();
+  const pathName = usePathname();
+  // is the connected user in a platform organization or not
+  const isPlatformUser = session?.data?.isPlatformUser ?? false;
```

Comment:
`getInitialProps` doesn't run on SSG pages AFAIU, so this isn't correctly set there.

https://www.loom.com/share/21300c444d934f63915f9c57f2325d52